content/enterprise_influxdb/v1.8/guides/replacing-nodes.md

```diff
@@ -69,14 +72,13 @@ The process of replacing both responsive and unresponsive data nodes is the same
 
 ### Reconnecting a data node with a failed disk
 
-A disk drive failing is never a good thing, but it does happen, and when it does,
-all shards on that node are lost.
+If a disk drive fails, all shards on that node are lost. Often in this scenario, you can replace the disk rather than replacing the entire host.
 
-Often in this scenario, rather than replacing the entire host, you just need to replace the disk.
-Host information remains the same, but once started again, the `influxd` process doesn't know
-to communicate with the meta nodes so the AE process can't start the shard-sync process.
+To restart AE and sync shards across the cluster, connect the `influxd` process running on the newly replaced disk to the cluster by doing one of the following:
 
-To resolve this, log in to a meta node and use the [`influxd-ctl update-data`](/enterprise_influxdb/v1.8/administration/cluster-commands/#update-data) command
+- For nodes with more than ≈20 GB of data on disk, ?
```

Comment:
what's our best practice here?

Reply:
@andrew-influx is it something like this?
· Make file copies of all shards to another location on network just in case
· influxd-ctl add-data
· manual shard rebalancing
· influxd-ctl copy-shard to copy shards to additional node
· influxd-ctl remove-shard to remove shards from node
· remove data node: influxd-ctl remove-data